src/utils/activityDetailsUtils.ts

```
@@ -38,9 +38,19 @@ export const getStatusBadgeConfig = (
+      bgColor: 'bg-sky-50',
+      textColor: 'text-sky-700',
+    },
+    [STUDENT_ACTIVITY_STATUS.GABARITO_RECEBIDO]: {
+      label: 'Gabarito Recebido',
+      bgColor: 'bg-emerald-50',
+      textColor: 'text-emerald-800',
+    },
   };
 
-  return configs[status];
+  return configs[status] ?? { label: status, bgColor: 'bg-gray-50', textColor: 'text-gray-700' };
```

ESLint

[error] Replace `configs[status]·??·{·label:·status,·bgColor:·'bg-gray-50',·textColor:·'text-gray-700'·}` with `(⏎····configs[status]·??·{⏎······label:·status,⏎······bgColor:·'bg-gray-50',⏎······textColor:·'text-gray-700',⏎····}⏎··)`

(prettier/prettier)